python/dgl/nn/pytorch/conv/egnnconv.py

```diff
@@ -0,0 +1,147 @@
+"""Torch Module for E(n) Equivariant Graph Convolution Layer"""
+# pylint: disable= no-member, arguments-differ, invalid-name
+import torch
+import torch.nn as nn
+
+class EGNNConv(nn.Module):
+    r"""E(n) Equivariant Convolutional Layer from `E(n) Equivariant Graph
+    Neural Networks <https://arxiv.org/abs/2102.09844>`__
+
+    .. math::
+        m_{ij}=\phi_e(h_i^l, h_j^l, ||x_i^l-x_j^l||^2, a_{ij})
+        x_i^{l+1} = x_i^l + C\sum(x_i^l-x_j^l)\phi_x(m_{ij})
```

Comment:
Should we change "\sum" to "\sum_{j\in\mathcal{N}(i)}" for the current implementation?